src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Web3
+description: What is web3 and why is it important?
+lang: en
+sidebar: true
+---
+
+# Introduction to web3 {#introduction}
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, these centralized entities have a stronghold on large swathes of the internet, undemocratically deciding what should and should not be allowed.
```

Comment:
> Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives.

Feels a bit strange to be saying "centralization" here. It's not very clear what that's referring to. Perhaps "centralized institutions have helped..."?